packages/expo-cli/src/commands/build-native/BuildError.ts

```diff
@@ -0,0 +1,12 @@
+import ExtendableError from 'es6-error';
+
+export default class BuildError extends ExtendableError {
```

Comment:
You are only throwing this error and never checking for it, so probably it would be better to just user Error